ewah/operators/s3.py

```diff
@@ -71,6 +83,7 @@ def __init__(
         decompress=False,
         file_load_parallelism=1000,
         thread_pool_size=20,
+        f_transform_datum=None,
```

Comment:
Comment (blocking): I am not a fan of introducing this here since I actually built this functionality in EWAH more generalized. I guess since EWAH is deprecated it's not a big deal... Yet if you can use cleaner_callables, that is already what you are looking for :)

Reply:
I decided to remove this transformation altogether from Python. It is generally simpler if we keep it in SQL, where we can also handle schema evolution more easily whenever needed.

Reply:
Sounds good :)